### ISSUE TITLE
Warn users of possible duplicate entries

### ISSUE THREAD

**Issue by literarymachine** [hbz/oerworldmap#848]
As an editor, I want to be notified that a new entry I am creating might be a duplicate of an existing one, e.g. if the `URL` or `name` are already present in the database.

**Comment by acka47** [comment]
To search for possible duplicate entries @karindr needed auto truncation. As this will be switched off (see #1026 ) we need another (better) way to warn of possible duplicates.

**Comment by karindr** [comment]
Will this be part of the new UI? We (@acka47 , @trugwaldsaenger and @philboeselager)  talked about it today and agreed that it would be super helpful.

**Comment by karindr** [comment]
Again we have a duplicate entry made by a user. it is always  quite some work to make one entry out of two and to mail to the users to have a look at it. So it would be really helpful to prevent duplicates.

**Comment by karindr** [comment]
On production I can get rid of the list by clicking on "OER World Map" on beta the list stays there.
On production the map zoomes out when you click on "OER World Map" on beta the zoom level says the same.

**Comment by karindr** [comment]
What does the little arrow button next to the plus and minus botton means/does?

**Comment by trugwaldsaenger** [comment]
> What does the little arrow button next to the plus and minus botton means/does?

You can move the map with this, even in a more horizontal position - pretty cool...

**Comment by trugwaldsaenger** [comment]
> On production I can get rid of the list by clicking on "OER World Map" on beta the list stays there.
On production the map zoomes out when you click on "OER World Map" on beta the zoom level says the same

@karindr Is this related to this issue? Or did you put it here by accidend?

**Comment by karindr** [comment]
Sorry wrong issue. Copy it to #194

**Comment by trugwaldsaenger** [comment]
@literarymachine: I agree that this is a very important function - we should keep care of it after the relaunch ASAP.

**Comment by karindr** [comment]
Again we have two new duplicates. Even OEGlobal was submitted again!

**Comment by literarymachine** [comment]
> Again we have two new duplicates. Even OEGlobal was submitted again!

Yes, one of them was me while demonstrating to someone. Just out of interest, do you have actual numbers how often this happens?

**Comment by karindr** [comment]
About twice a month I would say. It creates a lot of work for me because I have to look at both entries and collect the better information in one profile and delete the second. Sometimes I contact the users to explain to them why I will delete their entry and to have a look at the new entry. It is not very encouraging when your first contibution to a platform where you have voluntarily contributed something is deleted, so it would be great if we could prevent people from doing duplicates.

**Comment by literarymachine** [comment]
For now, check `name` while it is being entered, not restricted to the current type.

**Comment by karindr** [comment]
This time we had 3 times OERcamp 18 Nord and 2 times OERcamp 18 West

**Comment by trugwaldsaenger** [comment]
Just saw that slidewiki was added twice:
https://oerworldmap.org/resource/?q=slidewiki&filter.about.@type=Service&size=20&features=true
@karindr, can you have a look at it?

**Comment by karindr** [comment]
Done.
